src/App.vue

```diff
@@ -109,6 +113,14 @@ export default class App extends Mixins(TransactionMixin, LoadingMixin) {
   }
 
   goTo (name: PageNames): void {
+    if (name === PageNames.Wallet && this.currentRoute !== WALLET_DEFAULT_ROUTE) {
+      this.navigate({ name: WALLET_DEFAULT_ROUTE })
+    }
+
+    this.changePage(name)
+  }
+
+  changePage (name: PageNames): void {
```

Comment:
`changePage` can be `private` method right?

Reply:
I mean if it's not used in template we can make it `private`

Reply:
@stefashkaa Good point, thanks!